packages/backend/src/accounting/psql/ledger-transfer/model.ts

```diff
@@ -41,6 +41,22 @@ export class LedgerTransfer extends BaseModel {
     this.transferRef = this.transferRef || uuid()
   }
 
+  public get isPending(): boolean {
+    return this.state === LedgerTransferState.PENDING
+  }
```

Comment:
:thinking: should this return false if expired?

Reply:
Hmm, I might just remove this method and do
```ts
  public get expired(): boolean {
    return (
      this.state === LedgerTransferState.PENDING &&
      !!this.expiresAt &&
      this.expiresAt <= new Date()
    )
  } 
```